packages/pure-markdown/README.md

```diff
@@ -0,0 +1,3 @@
+# @khanacademy/pure-markdown
+
+An extended version of `@khanacademy/simple-markdown`.
```

Comment:
`pure-markdown` is a Node-compatible wrapper around `simple-markdown`. The [`index.ts`](https://github.com/Khan/perseus/blob/4e2b51002e41c8f36051775a7ceff4fd1a564526/packages/pure-markdown/src/index.ts#L2) from this file explains further: 

```ts
 * Contains markdown related functions in pure javascript,
 * extracted from perseus-markdown.jsx
 * Note that this file may be used in stand alone nodejs, thus
 * do not import anything from Perseus
```